cmd/2019_7/main_test.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/afarbos/aoc/pkg/utils"
+)
+
+func BenchmarkMain(b *testing.B) {
+	utils.EnableTestMain(&flagInput)
+
+	for i := 0; i < b.N; i++ {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)